chartfx-math/src/main/java/de/gsi/math/MathBasePrecise.java

```diff
@@ -0,0 +1,15 @@
+package de.gsi.math;
+
+/**
+ * Provide access to basic math routines with improved/guaranteed precision.
+ * // TODO: implement e.g. using apache commons math's PreciseMath
+ *
+ * @author akrimm
+ * @deprecated not implemented yet
+ */
+@Deprecated
+public class MathBasePrecise extends MathBase {
```

Comment:
Codacy found an issue: [Class cannot be instantiated and does not provide any static methods or fields](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)